[TR] Translation of index.md

mekici opened the pull request.

Atcold commented:
We already have translated `index.md`. What is this PR about?

Atcold commented:
Oh… I think you need to rebase on top of `master`.

Atcold commented on docs/tr/index.md:
```diff
@@ -1,6 +1,10 @@
 ---
 layout: default
+<<<<<<< HEAD
```
Garbage to take care of.

Atcold commented on docs/tr/index.md:
```diff
@@ -1,6 +1,10 @@
 layout: default
+<<<<<<< HEAD
+title: pytorch ile derin öğrenme
+=======
```
More garbage.

Atcold commented on docs/tr/index.md:
```diff
@@ -1,6 +1,10 @@
+title: pytorch ile derin öğrenme
+=======
 title: DERIN ÖĞRENME
+>>>>>>> b75959425fbd710c8e11004f6c91905343f98c78
```
More here.

Atcold commented on docs/tr/index.md:
```diff
@@ -1,6 +1,10 @@
+<<<<<<< HEAD
+title: pytorch ile derin öğrenme
+=======
 title: DERIN ÖĞRENME
```
This is the correct title.

Atcold commented on docs/tr/index.md:
```diff
@@ -236,6 +240,82 @@ Bu kurs,gözetimli ve gözetimsiz derin öğrenme, gömme yöntemleri, ölçev 
 
 
 
+<<<<<<< HEAD
```
Garbage.

Atcold commented on docs/tr/index.md:
```diff
@@ -236,6 +240,82 @@ Bu kurs,gözetimli ve gözetimsiz derin öğrenme, gömme yöntemleri, ölçev 
+    </tr>
+  </tbody>
+</table>
+=======
```
Garbage.

Atcold commented on docs/tr/index.md:
```diff
@@ -251,3 +331,4 @@ Bu kurs,gözetimli ve gözetimsiz derin öğrenme, gömme yöntemleri, ölçev 
 |Grader|<img src="https://st3.depositphotos.com/13159112/17145/v/450/depositphotos_171453724-stock-illustration-default-avatar-profile-icon-grey.jpg" width="100" height="100">|Pedro Vidal<br>pmh314@nyu.edu|
 |Grader|<img src="https://st3.depositphotos.com/13159112/17145/v/450/depositphotos_171453724-stock-illustration-default-avatar-profile-icon-grey.jpg" width="100" height="100">|Bixing Yan <br>by783@nyu.edu|
 
+>>>>>>> b75959425fbd710c8e11004f6c91905343f98c78
```
Garbage.

Atcold commented:
`index.md` is still polluted (from a failed merge conflict) and there is still an edit to the website format.

![image](https://user-images.githubusercontent.com/2119355/85928934-e4aaa480-b87e-11ea-9dec-2a6661d0c38d.png)

Atcold commented on docs/tr/index.md:
```diff
@@ -1,10 +1,10 @@
 ---
 layout: default
-title: DERIN ÖĞRENME
+title: Pytorch ile derin öğrenme
```
Revert.

Atcold commented on docs/tr/index.md:
```diff
@@ -235,19 +236,78 @@ Bu kurs,gözetimli ve gözetimsiz derin öğrenme, gömme yöntemleri, ölçev 
 
 
 
+## Ekip
```
I think the issue is with the table above. There is something that breaks all the table down.
Have you tried running the website locally?

mekici commented:
Hello Alfredo,

I am so sorry about the conflicted merge. I changed the table before and it
looks fine even in your docs folder. Please take a look:
https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/tr/index.md
But
somehow on this link https://atcold.github.io/pytorch-Deep-Learning/tr/ an
older version of the file is running. The one in your docs folder has a
table for people (please see the page source) but the one in use not. This
is why the people and the lecture table are mixed up. I reverted the commit
changing the layout.


On Mon, Jun 29, 2020 at 11:41 PM Alfredo Canziani <notifications@github.com>
wrote:

> *@Atcold* requested changes on this pull request.
>
> [image: Screenshot 2020-06-29 at 16 37 28]
> <https://user-images.githubusercontent.com/2119355/86053948-49086800-ba27-11ea-94f0-da31922fe6fc.png>
>
> You're still changing the website layout. Please, revert this.
> ------------------------------
>
> In docs/tr/index.md
> <https://github.com/Atcold/pytorch-Deep-Learning/pull/361#discussion_r447239055>
> :
>
> > @@ -1,10 +1,10 @@
>
>  ---
>
>  layout: default
>
> -title: DERIN ÖĞRENME
>
> +title: Pytorch ile derin öğrenme
>
>
> Revert.
> ------------------------------
>
> In docs/tr/index.md
> <https://github.com/Atcold/pytorch-Deep-Learning/pull/361#discussion_r447240530>
> :
>
> > @@ -235,19 +236,78 @@ Bu kurs,gözetimli ve gözetimsiz derin öğrenme, gömme yöntemleri, ölçev
>
>
>
>
>
>
>
> +## Ekip
>
>
> I think the issue is with the table above. There is something that breaks
> all the table down.
> Have you tried running the website locally?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Atcold/pytorch-Deep-Learning/pull/361#pullrequestreview-439482099>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADY7BDZNNF6S4OEIRT63ASTRZD365ANCNFSM4OAO5Q5A>
> .
>

Atcold commented:
> Please take a look: https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/tr/index.md

This *is* the file that *jekyll* is trying to convert to `HTML`, but there are errors, as you can see from the homepage. It is *not* using an older version. It is using the same version. *Jekyll* is reading the `MD` file but then fails to generate the conversion because of some issues with the table formatting. The fact that GitHub is able to visualise it correctly doesn't mean the file does not have errors.

Can you plz try to render the website locally and see where is the bug?

Atcold commented:
Any update here?

mekici commented:
Dear Alfredo,

I removed the fork and tried to push it again. I hope it works now.  I
tested the file locally before pushing.  Please check it and let me know if
there is any problem. Take care!

[image: Screen Shot 2020-07-17 at 17.59.38.png]

Regards,
_________________________________________________



Dr. Murat EKİCİ, Ph.D.
Usak University
Faculty of Education
Department Of Computer Education and Instructional Technology, Office #329
USAK/TURKEY



On Fri, Jul 17, 2020 at 2:43 AM Alfredo Canziani <notifications@github.com>
wrote:

> Any update here?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Atcold/pytorch-Deep-Learning/pull/361#issuecomment-659737475>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADY7BD75AXENSGBNBYVSD63R36GCDANCNFSM4OAO5Q5A>
> .
>

Atcold commented:
Did you create a *new* PR? Is that what you meant?
Here I still see the change in the submodule.

mekici commented:
Sorry for the belated response I was on vacation.  Yes, I created a new
pull request (#420 ) and everything seems to be working and aligned.

Regards,
_________________________________________________



Dr. Murat EKİCİ, Ph.D.
Usak University
Faculty of Education
Department Of Computer Education and Instructional Technology, Office #329
USAK/TURKEY



On Sun, Jul 19, 2020 at 7:56 AM Alfredo Canziani <notifications@github.com>
wrote:

> Did you create a *new* PR? Is that what you meant?
> Here I still see the change in the submodule.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Atcold/pytorch-Deep-Learning/pull/361#issuecomment-660587411>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADY7BDZVAJVIO2HFYCZEMXDR4J4F7ANCNFSM4OAO5Q5A>
> .
>

Atcold commented:
If you are okay, please close the PR.
